lib/mix/tasks/phx.gen.html.ex

```diff
@@ -2,54 +2,62 @@ defmodule Mix.Tasks.Phx.Gen.Html do
-  Elixir struct. It is followed by an optional list of attributes,
-  with their respective names and types. See `mix phx.gen.schema`
-  for more information on attributes.
-
-  Overall, this generator will add the following files to `lib/`:
-
-    * a context module in `lib/app/accounts.ex` for the accounts API
-    * a schema in `lib/app/accounts/user.ex`, with an `users` table
-    * a controller in `lib/app_web/controllers/user_controller.ex`
-    * an HTML view collocated with the controller in `lib/app_web/controllers/user_html.ex`
-    * default CRUD templates in `lib/app_web/controllers/user_html`
-
-  ## The context app
-
-  A migration file for the repository and test files for the context and
-  controller features will also be generated.
-
-  The location of the web files (controllers, HTML views, templates, etc) in an
-  umbrella application will vary based on the `:context_app` config located
-  in your applications `:generators` configuration. When set, the Phoenix
-  generators will generate web files directly in your lib and test folders
-  since the application is assumed to be isolated to web specific functionality.
-  If `:context_app` is not set, the generators will place web related lib
-  and test files in a `web/` directory since the application is assumed
-  to be handling both web and domain specific functionality.
-  Example configuration:
-
-      config :my_app_web, :generators, context_app: :my_app
+  > over distinct contexts (e.g., `Accounts.User` and `Payments.User`).
+
+  This generator adds the following files to `lib/`:
+
+    * a controller in `lib/my_app/controllers/user_controller.ex`
+    * default CRUD HTML templates in `lib/my_app_web/controllers/user_html`
+    * an HTML view collocated with the controller in `lib/my_app_web/controllers/user_html.ex`
+    * a schema in `lib/my_app/accounts/user.ex`, with an `users` table
+    * a context module in `lib/my_app/accounts.ex` for the accounts API
+  
+  Additionally, this generator creates the following files:
+    
+    * a migration for the schema in `priv/repo/migrations`
+    * a controller test module in `test/my_app/controllers/user_controller_test.exs`
+    * a context test module in `test/my_app/accounts_test.exs`
+    * a context test helper module in `test/support/fixtures/accounts_fixtures.ex` 
+  
+  If the context already exists, this generator injects functions for the given resource into
+  the context, context test, and context test helper modules.
+
+  ## Umbrella app configuration
+
+  By default, Phoenix injects both web and domain specific functionality into the same
+  application. When using umbrella applications, those concerns are typically broken
+  into two separate apps, your context application - let's call it `my_app` - and its web
+  layer, which Phoenix assumes to be `my_app_web`.
+
+  You can teach Phoenix to use this style via the `:context_app` configuration option
```

Comment:
Small thing here, "teach" implies that Phoenix is capable of learning. I'd change it to something like:

`Phoenix can be configured to use this style via the...`